First two blogposts

fleurlemire opened the pull request:
Hi Hannes,

I added the first two blogposts in the branch. I found it difficult what exactly was expected of the text in terms of language and content, but I did what I thought based on the issue. I'd like to hear what you think about this and whether this is somewhat in line with what you had in mind.

I made one post like you asked about:
- This is our blog and what we plan to do; a first introduction to the blog and what they can expect here
- Then a post on: we've been busy last semester developing content about causal inference (linkt o a few posts) and something else, in this case VM's since Fernando and Diego worked on that last semester (link to that too).

I am looking forward to hear what you think and what I could improve!

hannesdatta reviewed (changes requested):
Hi @fleurlemire - good for a start.

But the content is waaaaaay to generic and ChatGPT-ish!

Note that this is meant to be a genuine account of who we are, what we do, why we do it, and how we do it. 

I suggest to tone down big language ("like introduction to a window into our community and work" - that's way too "big"). The same holds for many other things that are being said, like "collective knowledge of our team". It's not like you're saying it, it just sounds over the top.

Try to find your own tone of voice, your own way to tell our story.

Can you revise?

Thanks.

fleurlemire commented:
Hi Hannes,

I changed the tone of voice and language of the posts. 
However, I am not sure what you mean with this and are therefore not sure what to do besides changing the text: _Note: I have fixed one code file, so when you push your revised version to the branch, the preview should actually render properly._
